src/app/results/results.component.html

```diff
@@ -50,6 +50,7 @@
             </div>
         </div>
         <br>
+        <div class="clean"></div>
```

Comment:
there are css for "clean" class it cleans styles that inherits from other styles like float 👍